src/table-list-field/table-list-field.component.ts

```diff
@@ -63,6 +63,8 @@ export class TableListFieldComponent extends AbstractListFieldComponent implemen
 
   onFieldAdd(field: string) {
     this.keys.push(field);
+    setTimeout(() => {
```

Comment:
OK, but since this function doesn't access the DOM, why do we wait for component to be created. Can't we just set its `path` and `tabIndex` to the map and then the new component can access it during initialisation.